docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -47,18 +47,18 @@ For the runtime libraries:
   ./emsdk install 2.0.33
   ./emsdk activate 2.0.33
   ```
-- Run `build nativeaot+libs -c [Debug|Release] -a wasm -os Browser`. This will create the architecture-dependent libraries needed for linking and runtime execution, as well as the managed binaries to be used as input to ILC.
+- Run `build build clr.nativeaotlibs+libs -c [Debug|Release] -a wasm -os Browser`. This will create the architecture-dependent libraries needed for linking and runtime execution, as well as the managed binaries to be used as input to ILC.
```

Comment:
```suggestion
- Run `build clr.nativeaotlibs+libs -c [Debug|Release] -a wasm -os Browser`. This will create the architecture-dependent libraries needed for linking and runtime execution, as well as the managed binaries to be used as input to ILC.
```